<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="aEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>65</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 85, 127);
color: rgb(233, 255, 37);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="resultLabel"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>2</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>7</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>9</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>0</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>5</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>8</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>3</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>1</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>4</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>6</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">adadGroup</string>
          </attribute>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="calcButton">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="jamButton">
          <property name="text">
           <string>+</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">operatorGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tafrighButton">
          <property name="text">
           <string>-</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">operatorGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="taghsimButton">
          <property name="text">
           <string>/</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">operatorGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="zarbButton">
          <property name="text">
           <string>*</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">operatorGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>379</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="adadGroup"/>
  <buttongroup name="operatorGroup"/>
 </buttongroups>
</ui>
